src/apiClients/stravaClient/stravaClient.ts

```diff
@@ -49,6 +51,11 @@ export class StravaClient {
         }
       );
 
+      if (process.env.NEXT_PUBLIC_INCLUDE_TEST_ITEM === 'true') {
+        response.data.unshift(
+          mockStravaActivity as unknown as StravaActivityResponse
```

Comment:
I'm not sure I understand what this does. Why can't you `mockStravaActivity as StravaActivityResponse` without first asserting that it's `unknown`?

Reply:
Short answer is that if TS doesn't think two types are actually compatible, it'll force you to convert to unknown as an intermediate to acknowledge that you're doing something bad

```ts
const a = 1 as string;
// Conversion of type 'number' to type 'string' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.(2352)

const a = 1 as unknown as string;
// okay
```